.pre-commit-config.yaml

```diff
@@ -17,6 +17,11 @@ repos:
       args: [ --fix ]
     # Run the formatter.
     - id: ruff-format
+- repo: https://github.com/pre-commit/mirrors-mypy
+  rev: 'v1.10.0'  # Use the sha / tag you want to point at
```

Comment:
Can we add a comment above this to say something like we do for Ruff above? `  # Ruff version. If bumping this, bump requirements-dev.txt as well`

Reply:
Added the comment.